modules/3d-tiles/src/lib/utils/s2/s2-bv-obb.ts

```diff
@@ -0,0 +1,64 @@
+import {getS2CellFromToken, get2dRegionFromS2Cell} from './s2-utils-ext';
+import {Vector3} from '@math.gl/core';
+import {OrientedBoundingBox, makeOrientedBoundingBoxFromPoints} from '@math.gl/culling';
+
+import {Ellipsoid} from '@math.gl/geospatial';
+
+export type S2BoundingVolume = {
+  token: string; // This can be an S2 key or token
+  minimumHeight: number;
+  maximumHeight: number;
+};
+
+function convertCartToXYZ(longitude, latitude, height, result?) {
```

Comment:
- What is `Cart`? Please type out. `LngLat`? `Cartographic`?
- Add JSDoc line
- Add typescript types